### PR TITLE
feat(MessageTypes): add CONTEXT_MENU_COMMAND (23)

### PR DESCRIPTION
**Pull request by advaith1**
**Please describe the changes this PR makes and why it should be merged:**
- https://github.com/discord/discord-api-docs/pull/3651

this might not actually ship

**Status and versioning classification:**
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by kyranet — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

File: src/util/Constants.js [hunk 1]
```diff
@@ -374,6 +374,7 @@ exports.InviteScopes = [
  * * APPLICATION_COMMAND
```
Comment (ImRodry) [inline src/util/Constants.js:374]: Shouldn't this be renamed to SLASH_COMMAND like it was on the upstream PR? Now that context menus are a thing, having an APPLICATION_COMMAND message type can be quite vague

Comment (ckohen) [reply src/util/Constants.js:374]: That would be a breaking change, it'll have to wait for v14

Comment (ImRodry) [reply src/util/Constants.js:374]: pretty sure there was a PR before v13 to change that, why wasn't that merged 🤔

**Review by vladfrangu — APPROVED** [review]

File: src/util/Constants.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iCrawl) [reply src/util/Constants.js:374]: Separate PR

**Pull request merged**